### DISCUSSION TITLE
Configurable Shears looting enchantment

### DISCUSSION THREAD

**Discussion by KioProject123** [PurpurMC/Purpur#943] [Feature Requests]
I want to turn off looting the snowman pumpkin. 
The snowman has only one pumpkin head, but you can looting multiple pumpkins, that's not reasonable :(

And if enable pumpkin-can-be-added-back, Then players can dupe pumpkins.

**Comment by BillyGalbreath** [comment]
If you want to turn off shearing the pumpkin, disable this option in purpur.yml

https://purpurmc.org/docs/Configuration/#drops-pumpkin-when-sheared

**Reply by KioProject123** [reply]
Thanks, I tried, but it breaks vanilla functionality. Maybe I can only use plugins to solve this situation.

In addition, the content of the document seems to be wrong, 
It should be "drop-pumpkin-when-sheared", not "drops-pumpkin-when-sheared", no S. lol
